tests/test_ruby.py

```diff
@@ -90,6 +90,11 @@ def test_rails_hello_world(auto_container_per_test):
 
 @pytest.mark.skipif(OS_VERSION != "tumbleweed", reason="no rails for ruby 2.5")
 def test_rails_template(auto_container_per_test):
+    # Rails asset pipeline needs Node.js and yarn
+    auto_container_per_test.connection.run_expect(
+        [0], "zypper -n in nodejs-default yarn libyaml-devel"
```

Comment:
you don't need nodejs & yarn
```suggestion
        [0], "zypper -n in ruby-devel libyaml-devel"
```

Reply:
so on the suggestion: ruby-devel should be already part of the container, if it isn't then that should be fixed first. a devel container for ruby without ruby-devel doesn't make any sense. 

and rails is printing warnings if it cannot compile the asset pipeline

Reply:
No idea how I came to the conclusion that installing `ruby-devel` would help in any way, because that package is more or less empty and only pulls in `ruby$version-devel` (but that one is already installed) :facepalm: